.github/scripts/generate_ascii_report.py

```
@@ -0,0 +1,83 @@
+│ Workflow    │ Before   │ After    │ Trend │ Performance             │
+├─────────────┼──────────┼──────────┼───────┼─────────────────────────┤""".format(
+        before_sparkline=generate_sparkline(data['baseline']),
+        before_avg=sum(data['baseline'])/len(data['baseline']),
+        after_sparkline=generate_sparkline(data['current']),
+        after_avg=sum(data['current'])/len(data['current'])
+    )
+
+    for name, metrics in data['workflows'].items():
+        change = ((metrics['before'] - metrics['after']) / metrics['before']) * 100
+        bar = generate_bar_chart(metrics['after'], 6)
+        report += f"\n│ {name:<11} │ {metrics['before']:>6.1f}m │ {metrics['after']:>6.1f}m │   {metrics['trend']}   │ {bar} │"
```

Pylint (3.3.7)

[convention] Line too long (127/100)

(C0301)